types/meteor/test/react-meteor-data-tests.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react/index';
```

Comment:
Why? You should just be able to do `import * as React from "react"`

Reply:
Fixed